### ISSUE TITLE
how would i play unloaded? 

### ISSUE THREAD

**Issue by LEVIXDDLMAO** [realDunkin/fnfmegapack-html5-test#1]

**Comment by realDunkin** [comment]
Go to freeplay and press tab and look for "VS Online" and press enter, there you can find unloaded and play it

**Comment by LEVIXDDLMAO** [comment]
oh okay are you fonna add things for the relase?

**Comment by realDunkin** [comment]
Yeah i have the assets and everything, I just need to code in the stage and fix stuff

**Comment by LEVIXDDLMAO** [comment]
oh okay and is there way to change notes in the realse? sorry i cant spell

**Comment by realDunkin** [comment]
Yeah there will be, I just have to make them work with 9 key and pixel versions aswell. Also you can change the color like in psych engine, i just need to make the menu

**Comment by LEVIXDDLMAO** [comment]
ah how long would that take?

**Comment by realDunkin** [comment]
I'm planning to release the update in April, but I'll definitly have it done by May

**Comment by LEVIXDDLMAO** [comment]
next year april?

**Comment by realDunkin** [comment]
No, this year lmao

**Comment by LEVIXDDLMAO** [comment]
sorry i thought april pasted

**Comment by realDunkin** [comment]
It's okay

**Comment by LEVIXDDLMAO** [comment]
ah okay

**Comment by realDunkin** [comment]
I have been taking a long time with updates, mostly because I have school

**Comment by LEVIXDDLMAO** [comment]
yeah i understand

**Comment by LEVIXDDLMAO** [comment]
Yo whats up

**Comment by LEVIXDDLMAO** [comment]
Oh i came up with an idea

**Comment by LEVIXDDLMAO** [comment]
yo

**Comment by realDunkin** [comment]
Yo

**Comment by LEVIXDDLMAO** [comment]
i have an idea

**Comment by realDunkin** [comment]
Oh what is it?

**Comment by LEVIXDDLMAO** [comment]
i could work on the realse version while you work on the beta?

**Comment by realDunkin** [comment]
Well the beta version is what the release version will become

**Comment by LEVIXDDLMAO** [comment]
yeah i know but is it yes or no? I need to get better at coding for my fnf engine witch is call Funked engine
